deno_webgpu/src/lib.rs

```diff
@@ -1,5 +1,7 @@
 // Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
 
+#![warn(unsafe_op_in_unsafe_fn)]
+
```

Comment:
@crowlKats: This is the only part of the review tagging you for code ownership. I'm assuming this change won't be controversial for you; if so, don't feel obligated to review this in its entirety. 🙂

Reply:
yea nothing wrong with this change :)